recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set name = "ESMValCore" %}
-{% set version = "2.10.0rc1" %}
+{% set package_name = "ESMValCore" %}
```

Comment:
Hi @bouweandela.  Ah, I see.  

Our thinking was that two "names" was a confusing factor for us. 

We'd read `name: {{ name|lower }}` as making the var (set) `name` now lowercase in future instances.  

Is there another way we could clarify this point?  And still keep with general conda-forge recipe practice?

Reply:
It would help if you read the documentation, then you can understand what is going on. Conda recipes use jinja2 templates, see https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#jinja-templates. You can find more information here: https://jinja.palletsprojects.com/en/3.0.x/templates/#filters

Reply:
Ok, thanks.  Done here: 1a3cefd.

Reply:
Thanks!